recipe/meta.yaml

```diff
@@ -56,8 +56,12 @@ outputs:
         - shapely >=1.8.4,<3.0.0
         # ipython extra
         - bigquery-magics >=0.6.0
+        # leaving out the matplotlib extra for now because it doesn't
+        # work with python 3.9 unless we move away from noarch: python
+        # - matplotlib-base >=3.7.1,<=3.9.2  # [py<310]
+        # - matplotlib-base >=3.10.3  # [py>=310]
```

Comment:
I'm having to leave out the matplotlib extra for now to keep `noarch: python`.  @conda-forge/google-cloud-bigquery, please let me know if you'd prefer to drop `noarch: python` (in a follow-up PR) instead.